addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -541,6 +541,9 @@
         <field name="mode">primary</field>
         <field name="priority">32</field>
         <field name="arch" type="xml">
+            <xpath expr="//tree" position="attributes">
+                <attribute name="default_order">date_from DESC</attribute>
+            </xpath>
```

Comment:
Is it needed? It's already the default order for Time Offs defined on the model, so the view should already order it that way

Reply:
I thought so too but it was in ascending order and worked when I added DESC here :/ I can show you

Reply:
According to JS framework team it's not working because the default order for group by (order in model is not for groups) is ascending and it needs to be specified explicity if i want it descending.